tests/integration_tests/functional/test_kernel_cmdline.py

```diff
@@ -26,6 +28,9 @@ def test_init_params(uvm_plain):
     )
 
     vm.start()
+    # Let the guest start booting for the serial output file to appear
+    time.sleep(1)
```

Comment:
In microvm.py, could we wait until the file exists? I'm not sure how long that takes and it would only affect microvms launched under screen, which are just a few.

Reply:
yeah, I was thinking about that too, let me see.

Reply:
done